lib/purescript-cst/src/Language/PureScript/CST/Parser.y

```diff
@@ -1,7 +1,6 @@
 {
 module Language.PureScript.CST.Parser
   ( parseType
-  , parseExpr
```

Comment:
What are the odds someone's using this to parse an expression somewhere?

Reply:
I wouldn't be surprised if it isn't used downstream, but in any case I think this is quite likely to be useful, and doesn't really cost us much to keep it aroundas it's just the `expr` declaration from the grammar, which has to exist anyway, so I'd be tempted to make an exception in this case.